pcsx2/gui/Panels/PathsPanel.cpp

```diff
@@ -79,3 +74,13 @@ Panels::StandardPathsPanel::StandardPathsPanel( wxWindow* parent )
 	GetSizer()->SetMinSize( wxSize( 475, GetSizer()->GetMinSize().GetHeight() ) );
 }
 
+void Panels::StandardPathsPanel::Apply()
+{
+
+}
+
+void Panels::StandardPathsPanel::AppStatusEvent_OnSettingsApplied()
+{
+
+}
```

Comment:
What purpose do these functions serve?

Reply:
This fixes the warning in first time wizard and the function to block users to not press next until they provide a valid bios.

Reply:
It relies on the fuction  onselbios, a cursor or atleast an update to the screen where you pick a bios.

So it handles this:
![image](https://user-images.githubusercontent.com/24227051/129057434-cfdaa323-d0f6-4d47-ade8-5f42f0f3bd80.png)

Whoever created this made it hardcoded and obfuscated.

Reply:
Yep, I can confirm those are necessary, they are assumed to be defined by inheritance. Other windows have them dummied out too; blame wx/air